### ISSUE TITLE
MSBuild project XML formatting is not preserved when loading / saving

### ISSUE THREAD

**Issue by tintoy** [NuGet/Home#4767]
## Details about Problem

NuGet product used: `dotnet` CLI.

dotnet --info:

```
.NET Command Line Tools (1.0.0-rc4-004771)

Product Information:
 Version:            1.0.0-rc4-004771
 Commit SHA-1 hash:  4228198f0e

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.0-rc4-004771
```

Worked before? If so, with which NuGet version:

As far as I know it's never worked.

## Detailed repro steps so we can see the same problem

1. `dotnet new console`
2. Reformat XML inside project file (maybe add a blank line, or put some attributes on a new line) and save it.
3. `dotnet add package Newtonsoft.Json`

## Expected behaviour

Formatting is preserved.

## Observed behaviour

Formatting is obliterated; even the blank lines present in the project after `dotnet new` are missing.

## Suggestions

It looks to me like this might be the culprit:

https://github.com/NuGet/NuGet.Client/blob/0c21c5c5066e07f997c5c891eead9965c04617c3/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs#L332

If you pass `preserveFormatting: true` to `ProjectRootElement.Open` then it should correctly retain the changes to the project from step 2.

**Comment by tintoy** [comment]
Bump, bump - anybody have time to look at this? I'd be willing to submit a PR if that would help. It's pretty annoying to use at the moment :-)

**Comment by tintoy** [comment]
Ah, I see - turns out this was independently discovered and fixed in #5697. Close enough :)
